arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,393 @@
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure SQL Database to retrieve links for."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ReplicationLinkListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ReplicationLinkProperties": {
+      "properties": {
+        "partnerServer": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the Azure SQL Server hosting the partner Azure SQL Database."
+        },
+        "partnerDatabase": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the partner Azure SQL Database."
+        },
+        "partnerLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Azure Region of the partner Azure SQL Database."
+        },
+        "role": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The role of the Azure SQL Database in the replication link.",
+          "enum": [
+            "Primary",
+            "Secondary",
+            "NonReadableSecondary",
+            "Source",
+            "Copy"
+          ],
+          "x-ms-enum": {
+            "name": "ReplicationRole"
+          }
+        },
+        "partnerRole": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The role of the partner Azure SQL Database in the replication link.",
+          "enum": [
+            "Primary",
+            "Secondary",
+            "NonReadableSecondary",
+            "Source",
+            "Copy"
+          ],
+          "x-ms-enum": {
+            "name": "ReplicationRole"
+          }
+        },
+        "startTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The start time for the replication link (ISO8601 format)."
+        },
+        "percentComplete": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The percentage of seeding complete for the replication link."
+        },
+        "replicationState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The replication state for the replication link.",
+          "enum": [
+            "PENDING",
+            "SEEDING",
+            "CATCH_UP",
+            "SUSPENDED"
```

Comment:
I actually have a link in CATCH_UP state right now, so I think yes to all

---
In reply to: [85475364](https://github.com/Azure/azure-rest-api-specs/pull/643#discussion_r85475364) [](ancestors = 85475364)